src/overview.md

```diff
@@ -0,0 +1,173 @@
+    // y ∈ <0,1>, the ratio of those inside a unit circle
+    // should approach π / 4. Therefore, the value of π
+    // should be:
+    yield ratio * 4;
+  }
+}
+Iterable<Point> generateRandom([int? seed]) sync* {
+  final random = Random(seed);
+  while (true) {
+    yield Point(random.nextDouble(), random.nextDouble());
+  }
+}
+class Point {
+  final double x, y;
+  const Point(this.x, this.y);
+  bool get isInsideUnitCircle => x * x + y * y <= 1;
+}
+```
+
+[This sample can be run
+live](https://dartpad.dev/b2e3a914bd39d0408d0580c42ef2e58b?null_safety=true)
+using [DartPad](https://dartpad.dev), our online execution environment for Dart.
+For more on the language, take the [Dart language
+tour](https://dart.dev/guides/language/language-tour).
+
+## Dart: The Libraries
+
+Dart has [a rich set of core libraries](https://dart.dev/guides/libraries) that
+provide essentials for many everyday programming tasks. This includes:
+
+- Built-in types, collections, and other core functionality for every Dart
+  program (`dart:core`)
+- Richer collections types such as queues, linked lists, hashmaps and binary
+  trees (`dart:collection`)
+- Encoders and decoders for converting between different data representations,
+  including JSON and UTF-8 (`dart:convert`)
+- Mathematical constants and functions, and random number generation
+  (`dart:math`)
+- File, socket, HTTP, and other I/O support for non-web applications (`dart:io`)
+- Support for asynchronous programming, with classes such as Future and Stream
+  (`dart:async`)
+- Lists that efficiently handle fixed sized data (for example, unsigned 8 byte
+  integers) and SIMD numeric types (`dart:typed_data`)
+- Foreign function interfaces for interop with other code that presents a
+  C-style interface (`dart:ffi`)
+- Concurrent programming using isolates: independent workers that are similar to
+  threads but don't share memory, communicating only via messages
+  (`dart:isolate`)
+- HTML elements and other resources for web-based applications that need to
+  interact with the browser and the DOM (Document Object Model) (`dart:html`)
+
+Our [library tour](https://dart.dev/guides/libraries/library-tour) demonstrates
+some of the classes provided in Dart through a series of worked examples.
+
+Beyond the core libraries, many APIs are provided through a comprehensive set of
+packages. Dart itself offers many useful supplementary packages including
+[characters](https://pub.dev/packages/characters),
+[intl](https://pub.dev/packages/intl), [http](https://pub.dev/packages/http),
+[crypto](https://pub.dev/packages/crypto) and
+[markdown](https://pub.dev/packages/markdown). In addition, there are many
+thousands of packages from third-party publishers and the broader community,
+including packages for [XML](https://pub.dev/packages/xml), [Windows
+integration](https://pub.dev/packages/win32),
+[SQLite](https://pub.dev/packages/sqflite_common) and [compression]
```

Comment:
The `compression` link is broken; I think it's the linefeed that needs to be removed